### PR TITLE
Add area delivery module check

### PR DESCRIPTION
**Pull request by marsender**
In getCheapestDelivery, the country area must be checked or we may select an incorrect delivery.
Most common case is probably the localdelivery, with a price of "0" for example. This delivery price is selected even if the customer country is not valid.
So I have added the same check as in the delivery selection file:
/vendor/thelia/core/lib/Thelia/Core/Template/Loop/Delivery.php
Thanks

### CODE REVIEW

File: core/lib/Thelia/TaxEngine/TaxEngine.php [hunk 1]
```diff
@@ -128,8 +128,10 @@ public function getDeliveryCountry()
                     $this->taxState = $currentDeliveryAddress->getState();
                 } else {
                     $customerDefaultAddress = $customer->getDefaultAddress();
-                    $this->taxCountry = $customerDefaultAddress->getCountry();
-                    $this->taxState = $customerDefaultAddress->getState();
+                    if (isset($customerDefaultAddress)) {
+											$this->taxCountry = $customerDefaultAddress->getCountry();
```
Comment (roadster31) [inline core/lib/Thelia/TaxEngine/TaxEngine.php]: Could you please use spaces instead of tabs for indentation ?

Comment (marsender) [reply core/lib/Thelia/TaxEngine/TaxEngine.php]: Yes of course. I've replaced tabs with spaces

**Comment by roadster31** [comment]
There are some problems with the new Travis CI configuration which have been fixed by #2422. Could you please rebase ?

**Comment by marsender** [comment]
The rebase is done. It's ok.

**Comment by roadster31** [comment]
Thanks 👍

**Pull request merged**